frontend/packages/console-shared/src/components/dashboard/resource-quota-card/AppliedClusterResourceQuotaItem.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  getQuotaResourceTypes,
+  QuotaScopesInline,
+  QuotaGaugeCharts,
+} from '@console/internal/components/resource-quota';
+import { ResourceLink } from '@console/internal/components/utils/resource-link';
+import { AppliedClusterResourceQuotaModel } from '@console/internal/models';
+import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
+
+import './resource-quota-card.scss';
+
+const AppliedClusterResourceQuotaItem: React.FC<AppliedClusterResourceQuotaItemProps> = ({
+  resourceQuota,
+  namespace,
+}) => {
+  const resourceTypes = getQuotaResourceTypes(resourceQuota);
+  const scopes = _.get(resourceQuota, 'spec.scopes');
+  return (
+    <>
+      <div>
+        <ResourceLink
+          kind={referenceForModel(AppliedClusterResourceQuotaModel)}
+          name={resourceQuota.metadata.name}
+          className="co-resource-item--truncate co-resource-quota-card__item-title"
+          namespace={namespace}
+          inline
+        />
+        {scopes && <QuotaScopesInline scopes={scopes} />}
+      </div>
+      <QuotaGaugeCharts
```

Comment:
I'd think we'd want to update `QuotaGuageCharts` to show both used for project and total used.